src/tests/ftest/harness/config.yaml

```diff
@@ -0,0 +1,8 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
```

Comment:
Just realized my latest change of `test_servers: 2` didn't get pushed... I'll need to update that and re-test before landing